symfony/plugins/orangehrmAdminPlugin/Api/EmailConfigurationAPI.php

```diff
@@ -291,13 +290,17 @@ public function setEmailConfiguration(EmailConfiguration $emailConfiguration)
 
             )
         );
-        $emailConfiguration->setSmtpPassword(
-            $this->getRequestParams()->getStringOrNull(
-                RequestParams::PARAM_TYPE_BODY,
-                self::PARAMETER_SMTP_PASSWORD
-
-            )
+        $password = $this->getRequestParams()->getStringOrNull(
+            RequestParams::PARAM_TYPE_BODY,
+            self::PARAMETER_SMTP_PASSWORD
         );
+        if ($password !== '********') {
```

Comment:
Please check when user input password as `********`, what should be the behaviour.
And I prefer to send this password as `null` from FE, if user not changing the password

Reply:
done